lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -147,7 +138,7 @@ def friendly_inspect
 Host:     #{host}
 Username: #{username}
 Database: #{database}
-FRIENDLY
+      FRIENDLY
```

Comment:
eh, this is auto reindent by rubymine :rofl:, next time I'll `git diff` before commit